File: src/main/java/kr/hahaha98757/zombiesaddon/commands/Commands.java
package kr.hahaha98757.zombiesaddon.commands;

import net.minecraftforge.client.ClientCommandHandler;

public class Commands {
    public static void registerAll() {
        ClientCommandHandler.instance.registerCommand(new CommandSLA());
        ClientCommandHandler.instance.registerCommand(new CommandZSV());
        ClientCommandHandler.instance.registerCommand(new CommandZSVLines());
        ClientCommandHandler.instance.registerCommand(new CommandPowerupPatterns());
        ClientCommandHandler.instance.registerCommand(new CommandWaveDelays());
        ClientCommandHandler.instance.registerCommand(new CommandAutoSplits());
        ClientCommandHandler.instance.registerCommand(new CommandZombiesAddon());
        ClientCommandHandler.instance.registerCommand(new CommandDE());
        ClientCommandHandler.instance.registerCommand(new CommandBB());
        ClientCommandHandler.instance.registerCommand(new CommandAA());
        ClientCommandHandler.instance.registerCommand(new CommandPR());
    }
}

File: src/main/kotlin/kr/hahaha98757/zombiesaddon/commands/Commands.kt
package kr.hahaha98757.zombiesaddon.commands

import net.minecraftforge.client.ClientCommandHandler

object Commands {
    fun registerCommands() {
        ClientCommandHandler.instance.registerCommand(CommandDE)
        ClientCommandHandler.instance.registerCommand(CommandBB)
        ClientCommandHandler.instance.registerCommand(CommandAA)
        ClientCommandHandler.instance.registerCommand(CommandPR)
        ClientCommandHandler.instance.registerCommand(CommandPowerupPatterns)
        ClientCommandHandler.instance.registerCommand(CommandSla)
        ClientCommandHandler.instance.registerCommand(CommandZombiesAddon)
        ClientCommandHandler.instance.registerCommand(CommandZsv)
        ClientCommandHandler.instance.registerCommand(CommandZsvLines)
        ClientCommandHandler.instance.registerCommand(CommandAutoSplits)
        ClientCommandHandler.instance.registerCommand(CommandZaDebug)
    }
}